packages/create-react-app/index.js

```diff
@@ -111,20 +147,22 @@ function createApp(name, verbose, version, template) {
   checkAppName(appName);
   fs.ensureDirSync(name);
   if (!isSafeToCreateProjectIn(root)) {
-    console.log('The directory ' + chalk.green(name) + ' contains files that could conflict.');
+    console.log(
+      'The directory ' +
+        chalk.green(name) +
+        ' contains files that could conflict.'
```

Comment:
I think the real solution is to use string templates. I'd be fine even with not breaking anything inside a string template at all, even code inside  `${}`. We just happen to do that now by accident, but it does mess up how the template looks. String templates are built for this kind of stuff though, and we can't know if you want this or not. It's too ambiguous.

Not sure what else to do. If you didn't want to this to go past 80 chars, what would you do? You'd break at least part of it onto new lines, but then only partially breaking the args gets really weird and starts being really inconsistent. If you really didn't want this to break, we have that rfc for ignoring the next line so you could add that comment.